main/webapp/modules/core/scripts/index.js

```diff
@@ -73,7 +73,7 @@ var dictionary = "";
 $.ajax({
 	url : "command/core/load-language?",
 	type : "POST",
-	async : false,
+	async : false, // FIXME: blocking
```

Comment:
Do you mean here that we shouldn't use a synch call?

Reply:
Correct. These all block user interactivity of the browser (and you'll see warnings on your Javascript console due to them).